lint_test/banned_external_code/id_ban/analysis_options.yaml

```
@@ -0,0 +1,11 @@
+analyzer:
+  plugins:
+    - ../../custom_lint
```

Doesn't seem to work correctly otherwise.
```suggestion
    - custom_lint
```

---

I'm pretty sure the issue is from [the analysis_options.yaml](https://github.com/invertase/dart_custom_lint/issues/148) file in the `lib/` folder. After deleting it the lints show up in the IDE